R/constants.R

```diff
@@ -240,6 +240,12 @@ aglu.AVG_WOOD_DENSITY_KGM3 <- 500 # In kg per m3
 # Carbon content of wood is about 50 percent across species
 aglu.AVG_WOOD_DENSITY_KGCM3 <- 250 # In kg carbon per m3
 
+# Carbon content adjustments from unmanaged to managed
+aglu.CVEG_MULT_UNMGDFOR_MGDFOR <- 0.5
```

Comment:
What are the units here? Fraction C in biomass?

Reply:
No, this is unitless--it's the conversion factor from unmanaged forest to managed forest, where the latter is understood to be forest not in logging rotation, and the latter is forest in logging rotation. The average vegetation biomass of the logged forest is assumed to be 50% of that of the unlogged forest (integrated over the rotation period). Using 50% under the assumption that the veg biomass of the logged forest over the rotation period can be approximated by a triangle.